src/mscorlib/src/System/Reflection/RuntimeMethodInfo.cs

```diff
@@ -41,7 +41,7 @@ internal INVOCATION_FLAGS InvocationFlags
                     //
                     // first take care of all the NO_INVOKE cases. 
                     if (ContainsGenericParameters ||
-                         ReturnType.IsByRef ||
```

Comment:
Do we need `IsByRefLike` condition here as well? Or are the ByRefLike returns handled somewhere else?

Reply:
That gets handled in the "else" branch (`RuntimeMethodHandle.GetSecurityFlags(this);`)